src/MOI_wrapper/indicator_constraints.jl

```diff
@@ -53,10 +53,22 @@ function MOI.get(
         SCIPgetVarsLinear(o, linear_cons),
         nvars,
     )
+
+    scalar = Ref(1.0)
+    constant = Ref(0.0)
+    orig_vars = []
+    for i in 1:nvars
+        @assert scalar.x == 1.0 
+        @assert constant.x == 0.0
+        var = Ref(vars[i])
+        @SCIP_CALL SCIPvarGetOrigvarSum(var, scalar, constant)
+        push!(orig_vars, var[])
+    end
```

Comment:
```suggestion
    orig_vars = map(1:nvars) do i
        var = Ref(vars[i])
        @SCIP_CALL SCIPvarGetOrigvarSum(var, scalar, constant)
        @assert scalar[] == 1.0 
        @assert constant[] == 0.0
        var[]
    end
```

Given that the code is repeated, might be worth putting in a separate function and call it at the two places